pkg/csi/service/node.go

```diff
@@ -1176,9 +1182,7 @@ func publishFileVol(
 	if params.ro {
 		mntFlags = append(mntFlags, "ro")
 	}
-	if cnstypes.CnsClusterFlavor(os.Getenv(csitypes.EnvClusterFlavor)) == cnstypes.CnsClusterFlavorGuest {
-		mntFlags = append(mntFlags, "hard")
-	}
+	mntFlags = append(mntFlags, "hard")
```

Comment:
Can you also include the rest of the default parameters mentioned in https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/1174/files PR? I can close that one.